solidity/test/integration/LlamaVesting.t.sol

```diff
@@ -3,46 +3,76 @@ pragma solidity 0.8.20;
 
 import {IntegrationBase} from 'test/integration/IntegrationBase.sol';
 
+import {console} from 'forge-std/console.sol';
+/**
+ * TODO: Add generic test to show the behaviour of vesting contract + llamaPay stream
+ *       At the beginning of the unlocking period: 0 tokens
+ *       Then limited by vesting contract
+ *       Then limited by llamaPay (after vesting period has ended)
+ */
+
 contract IntegrationLlamaVesting is IntegrationBase {
   uint256 internal _vestingStartTime;
 
   function setUp() public override {
     super.setUp();
-    _vestingStartTime = block.timestamp;
+    _vestingStartTime = _connextVestingWallet.start();
   }
 
-  function test_VestAndUnlock() public {
-    vm.prank(payer);
-    _llamaPay.depositAndCreate(TOTAL_AMOUNT, address(_connextVestingWallet), PAY_PER_SECOND);
-
+  function test_VestAndUnlock_LaunchDate() public {
     // Before the cliff
-    uint256 _timestamp = _connextVestingWallet.cliff() - 1;
-    uint256 _vestedAmount = (_timestamp - _vestingStartTime) * PAY_PER_SECOND / 1e2;
-
+    uint256 _timestamp = VESTING_START_DATE;
```

Comment:
ambiguity here: `VESTING_START_DATE` should be related to the llamapay vest (for wonderland, Aug 1 2022), the connext "vest" date should be tested around `NEXT_LAUNCH_DATE` (which is shared to all partners, Sep 5 2023)